addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -56,7 +57,7 @@ const LinkPopoverWidget = Widget.extend({
         }
         const clipboard = new _ClipboardJS(
             this.$copyLink[0],
-            {text: () => this.target.href} // Absolute href
+            {text: () => this.url} // Absolute href
```

Comment:
I doubt we want to change that and copy the url with `/@/` inside it..
The goal is to reuse the link in another link, and we don't want user to copy paste and use link with `/@/` inside it